server/endpoints/network/patterns.endpoint.js

```diff
@@ -34,6 +35,18 @@ const single = async (request, reply) => {
 /* * */
 
 const realtime = async (request, reply) => {
+	const currentArchiveIds = {};
+
+	const allArchivesTxt = await SERVERDB.client.get('archives:all');
+	const allArchivesData = JSON.parse(allArchivesTxt);
+
+	for (const archiveData of allArchivesData) {
+		const archiveStartDate = DateTime.fromFormat(archiveData.start_date, 'yyyyMMdd');
+		const archiveEndDate = DateTime.fromFormat(archiveData.end_date, 'yyyyMMdd');
+		if (archiveStartDate > DateTime.now() || archiveEndDate < DateTime.now()) continue;
+		else currentArchiveIds[archiveData.operator_id] = archiveData.id;
```

Comment:
Make this statement multi line for better readability

Reply:
Thanks! I left it like this because the code is the same as the one in [stops endpoint route handler](https://github.com/carrismetropolitana/api/blob/beta/server/endpoints/network/stops.endpoint.ts) and in the [RTEVENTS service](https://github.com/carrismetropolitana/api/blob/beta/server/services/RTEVENTS.js)

It's just copied and pasted in place 🙂. If you feel like contributing with your suggestion, I think it'd be best to address them in a separate PR, ensuring the changes are applied consistently across all instances of this snippet.

Thank you again for your suggestion!